blocks/inspector-controls/button-control/index.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
This component is not used anymore, right?

Reply:
You're right! I've removed it.